iOS/Flows/Home/Moments/Capture/PIP/PiPRecorder.swift

```diff
@@ -89,10 +88,9 @@ class PiPRecorder {
     
     func recordAudio(sampleBuffer: CMSampleBuffer) {
         guard self.isReadyToRecord, let assetWriter = self.frontAssetWriter else { return }
-        
-        if assetWriter.status == .unknown {
-            self.startWritingSession(with: assetWriter, and: sampleBuffer)
-        } else if assetWriter.status == .writing {
```

Comment:
This is the only functional change. Audio and video sample are coming on different threads, yet share an asset writer. If they happen nearly simultaneously, both would think they needed to start the session. The session would be started twice, and then an error is thrown because we're trying to write two different files to the same location.